src/main/java/com/nerodesk/takes/doc/TsDoc.java

```diff
@@ -70,17 +71,26 @@ public TsDoc(final Base bse) {
         this.base = bse;
     }
 
+    // @todo #94:30min I tried to get filename from file meta data and it
+    //  doesn't look clean but works for now. We have to either wait for impl.
+    //  from `Takes` framework or implement in this project class that is going
+    //  to do this in clean way.
     @Override
     public Take route(final Request req) throws IOException {
         final String file;
         final Href href = new RqHref(req).href();
-        final Iterator<String> param = href.param("file").iterator();
+        final String key = "file";
+        final Iterator<String> param = href.param(key).iterator();
         if (param.hasNext()) {
```

Comment:
Do we really need this `if/else`? `filename` always is passed as part of multipart request?